chimingxuan/client/src/client.cpp

```diff
@@ -0,0 +1,77 @@
+#include "ros/ros.h"
+#include "server/Register.h"
+#include "server/Unregister.h"
+#include "std_msgs/String.h"
+
+#include <sstream>
+
+#include <ctime>
+
+#include <signal.h>
+
+void Unregister(int sig){
+    ros::NodeHandle n;
+    ros::ServiceClient client_unregister = n.serviceClient<server::Unregister>("unregister");
```

Comment:
**[cpplint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br> Lines should be <= 80 characters long  [whitespace/line_length] [2]